src/app/issuer/components/collectionbadge-selection-dialog/collectionbadge-selection-dialog.component.spec.ts

```diff
@@ -0,0 +1,62 @@
+import { TestBed } from '@angular/core/testing';
+import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { CollectionBadgeSelectionDialog } from "./collectionbadgebadge-selection-dialog.component";
+import { RouterTestingModule } from "@angular/router/testing";
+import { BadgrCommonModule, COMMON_IMPORTS } from "../../../common/badgr-common.module";
+import { COMMON_MOCKS_PROVIDERS_WITH_SUBS } from "../../../mocks/mocks.module.spec";
+import { FormsModule } from "@angular/forms";
+
+describe('CollectionBadgeSelectionDialogComponent', () => {
+  let fixture;
+  let component;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        CollectionBadgeSelectionDialog
+      ],
+			imports: [
+				RouterTestingModule,
+				CommonModule,
+				BadgrCommonModule,
+				FormsModule,
+				...COMMON_IMPORTS,
+			],
+			providers: [
+				...COMMON_MOCKS_PROVIDERS_WITH_SUBS,
+			],
+      schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
+    }).compileComponents();
+    fixture = TestBed.createComponent(CollectionBadgeSelectionDialog);
+    component = fixture.debugElement.componentInstance;
+		component.resolveFunc = () => {};
+		component.open = false;
+		component.close = () => {};
+  });
+
+  it('should create a component', async () => {
```

Comment:
I agree that these tests arent great, i guess they are also generated. Would you mind if i create another ticket for testing the dialog instead of doing it in this PR?

Reply:
Sounds good